scripts/extract-l10n.sh

```diff
@@ -8,10 +8,10 @@
 #   - Update your code
 #   - Extract new strings and push to the .po files
 #
-# If you're running the script manually please make sure to expose
+# For the fully automated experience make sure to expose
 # the following variables to the environment:
 #   - GITHUB_TOKEN (to the github token of addons-robot,
-#                   talk to tofumatt or cgrebs)
+#                   talk to @diox or @muffinresearch)
 #   - TRAVIS_REPO_SLUG="mozilla/addons-server"
```

Comment:
Can't we provide this as a default in the file?

Reply:
As `TRAVIS_BRANCH` this is automatically present in travis

Reply:
I'm thinking for local use so we don't have to define extra variables to make it run.

Reply:
Yeah I'll make sure no variables are needed for manual usage